docs-2.0/reuse/source_manage-service.md

```diff
@@ -25,6 +39,33 @@ $ sudo /usr/local/nebula/scripts/nebula.service
 |`storaged`|Set the Storage Service as the target service.|
 |`all`|Set all the Nebula Graph services as the target services.|
 
+## Manage services with systemd
+
+For easy maintenance, Nebula Graph supports managing services with systemd. Starting, stopping, restarting, and viewing services with command `systemctl`.
+
+!!! note
+
+  After installing the Nebula Graph, the `.service` files required for systemd are in the directory `etc/unit` of installation directory.  The Nebula Graph installed with the RPM/DEB package automatically places the `.service` files into the directory `/usr/lib/systemd/system`, and the parameter `ExecStart` also automatically generated from the specified Nebula Graph installation path, so you can use the command `systemctl` directly.
+
+  Otherwise, users need to move the `.service` files manually into the directory `/usr/lib/systemd/system`, and modify the file path of parameter `ExecStart` in the `.service` files.
```

Comment:
```suggestion
  Otherwise, users need to move the `.service` files manually into the directory `/usr/lib/systemd/system`, and modify the file path of the parameter `ExecStart` in the `.service` files.
```